### PR TITLE
FAT-5606: Get linking rule with ID and Patch requests covered

### PR DESCRIPTION
**Pull request by hasanepam**
## Purpose
Covering Get linking rule with ID and Patch linking rule requests and improving Get linking rules request.

## Approach
Added respective test case scenarios and assertion for _autoLinkingEnabled_ field

### Learning
[https://issues.folio.org/browse/FAT-5606](https://issues.folio.org/browse/FAT-5606)

### CODE REVIEW

File: mod-entities-links/src/main/resources/spitfire/mod-entities-links/features/linkingRulesTest.feature [hunk 1]
```diff
@@ -14,3 +15,37 @@ Feature: linking-rules tests
     And assert karate.sizeOf(response[0].authoritySubfields) > 0
     And assert response[0].bibField != null
     And assert response[0].authorityField != null
+    And assert response[0].autoLinkingEnabled != null
+
+  @Positive
+  Scenario: Get instance to authority rules by id - Should return appropriate rule
+    Given path '/linking-rules/instance-authority/' + defauldLinkId
+    When method GET
```
Comment (psmagin) [inline mod-entities-links/src/main/resources/spitfire/mod-entities-links/features/linkingRulesTest.feature:23]: Please add negative scenario for 404 status

File: mod-entities-links/src/main/resources/spitfire/mod-entities-links/features/linkingRulesTest.feature [hunk 2]
```diff
@@ -14,3 +15,37 @@ Feature: linking-rules tests
...
+    Then status 200
+    And assert response != null
+    And assert response.bibField == 100
+    And assert response.authorityField == 100
+    And assert response.autoLinkingEnabled != null
+    And assert karate.sizeOf(response.authoritySubfields) > 0
+
+  @Positive
+  Scenario: Patch linking rule by id - Should update appropriate rule
+    Given path '/linking-rules/instance-authority/' + defauldLinkId
```
Comment (psmagin) [inline mod-entities-links/src/main/resources/spitfire/mod-entities-links/features/linkingRulesTest.feature:33]: Please add more scenarios:
postive:
trying to update all fields of a rule -> only autoLinkingEnabled is updated

negative: 
rule not found -> 404

**Review by psmagin — CHANGES_REQUESTED** [review]

File: mod-entities-links/src/main/resources/spitfire/mod-entities-links/features/linkingRulesTest.feature (hunk 1 above)
Comment (hasanepam) [reply mod-entities-links/src/main/resources/spitfire/mod-entities-links/features/linkingRulesTest.feature:23]: Ok

**Review by viacheslavkol — APPROVED** [review]
It'd be better to have all @Positive first and then all @Negative. Now the order is mixed

**Pull request merged**